paddle/fluid/operators/momentum_op.cc

```diff
@@ -98,7 +98,7 @@ The update equations are as follows:
 $$
 velocity = mu * velocity + gradient \\
 if (use\_nesterov):   \\
-  param = param - gradient * learning\_rate + mu * velocity * learning\_rate \\
+  param = param - (gradient + mu * velocity) * learning\_rate \\
 else:   \\
   param = param - learning\_rate * velocity. \\
```

Comment:
even basic momemtum seems wrong to me, mu * velocity doesn't multiply learning rate...:
```
# Momentum update
v = mu * v - learning_rate * dx # integrate velocity
x += v # integrate position
```